### ISSUE TITLE
Doesn't let me launch my game anymore.

### ISSUE THREAD

**Issue by ghost** [PrismLauncher/PrismLauncher#2658]
### Operating System

Windows

### Version of Prism Launcher

Prism Launcher 8.4

### Version of Qt

This program uses Qt version 6.6.3.

### Description of bug

It keeps saying looks like we could launch after 3 tries would you like to try again i try again and it still doesn't launch.

### Steps to reproduce

It keeps saying looks like we could launch after 3 tries would you like to try again i try again and it still doesn't launch.

### Suspected cause

It keeps saying looks like we could launch after 3 tries would you like to try again i try again and it still doesn't launch.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by KujouShiori** [comment]
I'm also having this issue saying that getting a Minencraft access token failed: ElotransFelingheteps://Appi. Minecrafts.Combe/Longcher/Logkin-Severeprid:

**Comment by Multidakman** [comment]
Same here

**Comment by KujouShiori** [comment]
Login was fine yesterday, today you logged in with the same issue, trying to remove the account and log back in prompted that getting a Minencraft access token failed

**Comment by KujouShiori** [comment]
I tried clearing DNS and repairing the hosts file and it didn't work

**Comment by sweetstar383** [comment]
I had the same problem as ManakaGekka where it seems to be linked to Minecraft accounts not linking to prism launcher. 
" failure to get the Minecraft access token" sort of prompt. I updated to the latest launcher and restarted my computer to make sure

**Comment by powermaker450** [comment]
Seems like it might be a service issue. Even normal Minecraft Launcher doesn't work

**Comment by KujouShiori** [comment]
It should be a problem with the service, I use the official launcher to prompt a network error

**Comment by jhw2167** [comment]
Also having the issue.

Found a reddit thread that said to invalidate the metadata cache (Home Screen -> Help -> clear metadata) but this did not work.

Since then I removed my Microsoft Account and attempted to re-add it, but received an error:

`
Failed to get Minecraft access token: Error transferring https://api.minecraftservices.com/launcher/login - server replied: 
`
Looks like its a general issue and will hopefully go away soon.
